test/conftest.py

```diff
@@ -80,6 +89,20 @@ def build_mode(request: pytest.FixtureRequest) -> str:
     return mode
 
 
+@pytest.fixture(autouse=True)
+async def print_scylla_log_filename(request: pytest.FixtureRequest) -> AsyncGenerator[None]:
+    """Print out a path to a ScyllaDB log.
```

Comment:
Why async?

Reply:
you're right, there is no reason to make it async.